palvelutarjotin/urls.py

```diff
@@ -37,7 +39,16 @@ def healthz(*args, **kwargs):
 def readiness(*args, **kwargs):
     return HttpResponse(status=200)
 
-
-urlpatterns += [path("healthz", healthz), path("readiness", readiness)]
+def version(*args, **kwargs):
+    response_json = {
+        "status": "ok",
+        "release": settings.APP_RELEASE,
+        "packageVersion": __version__,
+        "commitHash": settings.COMMITHASH.decode("utf-8"),
+        "buildTime": settings.APP_BUILDTIME.strftime("%Y-%m-%dT%H:%M:%S.000Z"),
```

Comment:
As it's [build time](https://dictionary.cambridge.org/dictionary/english/build-time) (i.e. two words rather than one word) `APP_BUILDTIME` [should be](https://peps.python.org/pep-0008/#constants) `APP_BUILD_TIME` according to PEP8 Style Guide:
> Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.

Reply:
fixed on #323